### PR TITLE
adding finn code style module for formatting

### PR DESCRIPTION
**Pull request by leftieFriele**
adding the FINN module and fixing contrib config

### CODE REVIEW

**Comment by leftieFriele** [comment]
@gregersrygg @sveisvei @phillipj can I haz merge?

File: .jshintrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "./node_modules/finn-js-code-style/.jshintrc",
+    "browser": false,
+    "node": true,
+    "maxstatements": 10
```
Comment (phillipj) [inline .jshintrc]: Is this `maxstatements` really needed? It doesnt seem to fail if I remove it locally..

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "Simple gulp plugin for the maven-deploy module",
   "main": "index.js",
   "scripts": {
-    "test": "./node_modules/.bin/mocha"
+    "test": "./node_modules/.bin/mocha && ./node_modules/.bin/finn-js-code-style index.js, sample/**"
```
Comment (phillipj) [inline package.json]: FYI `./node_modules/.bin` isnt neccessary here, as the npm client puts everything in `./node_modules/.bin` into `$PATH` when running these scripts

**Comment by phillipj** [comment]
Ahh, you're too patient @leftieFriele, thanks for reminding us :+1: You're aaaaalmost there :dancer:

File: .jshintrc (hunk 1 above)
Comment (gregersrygg) [inline .jshintrc]: 10 is stricter than 20 ;)

File: package.json (hunk 2 above)
Comment (gregersrygg) [inline package.json]: What @phillipj said. Also, the ',' after index.js shouldn't be there.

**Comment by gregersrygg** [comment]
Sorry, didn't see this pull-request. Looks like a good improvement, but there's a typo in the test-command in package.json. Remove the comma (and preferably the unnecessary path to finn-js-code-style), and this is good to go :+1: 

Btw. We don't use this anymore at FINN. Do you want your child back @leftieFriele?

**Comment by micha149** [comment]
Just for understanding: `finn-js-code-style` is only a jshint preset what people from FINN.no are using?

**Comment by leftieFriele** [comment]
yeah, that's what it is. you should probably just remove it.

On Wed, Jun 3, 2015 at 10:39 AM, Michael van Engelshoven <
notifications@github.com> wrote:

> Just for understanding: finn-js-code-style is only a jshint preset what
> people from FINN.no are using?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/micha149/gulp-maven-deploy/pull/7#issuecomment-108247822>
> .
>



-- 
<quote goes here>
-----------------------------------------------------------------------
Espen Dalløkken

**Comment by micha149** [comment]
But I like the code style changes made in the PR :smile:

**Pull request merged**